dev/up

```diff
@@ -12,9 +12,18 @@ if [[ "${OSTYPE}" == "darwin"* ]]; then
     fi
     if ! kotlinc -version &>/dev/null; then brew install kotlin; fi
     if ! swiftformat -version &>/dev/null; then brew install swiftformat; fi
+    if ! foundryup -version &>/dev/null; then  
+        # install foundry for tests that require mocking blockchain
+        curl -L https://foundry.paradigm.xyz | bash
+        # you made need to adjust this depending on which $SHELL you use 
+        source $HOME/.zshenv
```

Comment:
do you know if `source $PROFILE` would work here?

ref: https://raw.githubusercontent.com/foundry-rs/foundry/master/foundryup/install

 echo "Detected your preferred shell is $PREF_SHELL and added foundryup to PATH."
echo "Run 'source $PROFILE' or start a new terminal session to use foundryup."
echo "Then, simply run 'foundryup' to install Foundry."

I haven't tried it myself yet

Reply:
no, as its a different shell environment ...

```
❯ dev/up
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  2189  100  2189    0     0  14474      0 --:--:-- --:--:-- --:--:-- 14474
Installing foundryup...

Detected your preferred shell is zsh and added foundryup to PATH.
Run 'source /Users/studd/.zshenv' or start a new terminal session to use foundryup.
Then, simply run 'foundryup' to install Foundry.
dev/up: line 19: PROFILE: unbound variable
```

... but If anyone does the error I think its self-explanatory enough to resolve.